Resources/Prototypes/_RMC14/Reagents/elements.yml

```diff
@@ -401,6 +401,13 @@
           min: 12
       - !type:Drunk
         boozePower: 2
+      - !type:HealthChange
+        conditions:
+        - !type:ReagentThreshold
+          min: 30
+        damage:
+          groups:
+            Toxin: 0.5
```

Comment:
This is supposed to be OD 60, crit OD 100
PROPERTY_ALCOHOLIC = 5, PROPERTY_FUELING = 3, PROPERTY_OXIDIZING = 3, PROPERTY_FLOWING = 2